AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Storage/SQLite/Statement+AnyModel.swift

```diff
@@ -36,7 +36,7 @@ extension Statement {
             throw error
         }
 
-        let instance = try ModelRegistry.decode(modelName: modelType.modelName, from: jsonString)
+        let instance = try ModelRegistry.decode(modelName: modelSchema.name, from: jsonString)
```

Comment:
`modelType.modelName` to `modelSchema.name`.  I think this was done because the callers no longer passed the modelType, so this had to change because we are now passing modelSchema. I don't think it was broken before. Question for Dynamic Model types aka Flutter: does `modelType` have modelName?